packages/client/src/utils/index.ts

```diff
@@ -7,8 +7,12 @@ import { pkgVersion } from '../pkg';
 import { NowBuildError } from '@vercel/build-utils';
 import { VercelClientOptions, VercelConfig } from '../types';
 import { Sema } from 'async-sema';
-import { pathExists, readFile } from 'fs-extra';
+import { readFile } from 'fs-extra';
 import readdir from './readdir-recursive';
+import {
+  findConfig,
```

Comment:
this should probably be named `findMicrofrontendsConfig` in the package, but I think this makes the code easier to read below?

```suggestion
  findConfig as findMicrofrontendsConfig,
```